src/Servers/Kestrel/samples/Http3SampleApp/Program.cs

```diff
@@ -29,6 +29,8 @@ public static void Main(string[] args)
                         options.ConfigureHttpsDefaults(httpsOptions =>
                         {
                             httpsOptions.ServerCertificate = cert;
+                            // httpsOptions.ClientCertificateMode = ClientCertificateMode.AllowCertificate;
+                            // httpsOptions.AllowAnyClientCertificate();
```

Comment:
remove?

Reply:
I left them intentionally for anyone working with client certs.